source/includes/harvest/_users.md

```diff
@@ -76,12 +76,12 @@ id | The ID of the user whose future job permissions to retrieve
 
 An array of objects of the following:
 
-Parameter | Description
---------- | -----------
-id | The ID representing the future job permissions
-office_id | The ID of the office. The user role will be granted when a job is created belonging to this office.
-department_id | The ID of the department. The user role will be granted when a job is created belonging to this department.
-user\_role\_id | The ID of the user role that will be granted to the user.
+Parameter | Type | Description
+--------- | ----------- | ---------------
+id | integer | The ID representing the future job permissions
```

Comment:
What about "The unique ID associating a specific user to a future job permission."